### PR TITLE
Update nightlies 1.12->1.13

### PR DESCRIPTION
**Pull request by atalman**
Release 1.13 : Update nightlies 1.12->1.13

### CODE REVIEW

**Review by seemethere — CHANGES_REQUESTED** [review]
Shouldn't this be an update to 1.14?

File: .circleci/scripts/binary_ios_upload.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ fi
 cp ${PROJ_ROOT}/LICENSE ${ZIP_DIR}/
 # zip the library
 export DATE="$(date -u +%Y%m%d)"
-export IOS_NIGHTLY_BUILD_VERSION="1.12.0.${DATE}"
+export IOS_NIGHTLY_BUILD_VERSION="1.13.0.${DATE}"
```
Comment (malfet) [inline .circleci/scripts/binary_ios_upload.sh:36]:
```suggestion
export IOS_NIGHTLY_BUILD_VERSION="1.14.0.${DATE}"
```

File: .circleci/scripts/binary_populate_env.sh [hunk 2]
```diff
@@ -102,7 +102,7 @@ PIP_UPLOAD_FOLDER='nightly/'
 # We put this here so that OVERRIDE_PACKAGE_VERSION below can read from it
 export DATE="$(date -u +%Y%m%d)"
 #TODO: We should be pulling semver version from the base version.txt
-BASE_BUILD_VERSION="1.12.0.dev$DATE"
+BASE_BUILD_VERSION="1.13.0.dev$DATE"
```
Comment (malfet) [inline .circleci/scripts/binary_populate_env.sh:105]: Is this file even used for anything?
```suggestion
BASE_BUILD_VERSION="1.14.0.dev$DATE"
```

File: .circleci/scripts/binary_populate_env.sh [hunk 3]
```diff
@@ -165,7 +165,7 @@ if [[ "${BUILD_FOR_SYSTEM:-}" == "windows" ]]; then
 fi
 
 export DATE="$DATE"
-export NIGHTLIES_DATE_PREAMBLE=1.12.0.dev
+export NIGHTLIES_DATE_PREAMBLE=1.13.0.dev
```
Comment (malfet) [inline .circleci/scripts/binary_populate_env.sh:168]: Same as above
```suggestion
export NIGHTLIES_DATE_PREAMBLE=1.14.0.dev
```

File: android/gradle.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ABI_FILTERS=armeabi-v7a,arm64-v8a,x86,x86_64
 
-VERSION_NAME=1.12.0-SNAPSHOT
+VERSION_NAME=1.13.0-SNAPSHOT
```
Comment (malfet) [inline android/gradle.properties:3]:
```suggestion
VERSION_NAME=1.14.0-SNAPSHOT
```

**Review by malfet — COMMENTED** [review]
I'm really confused about this one. 
a) Shouldn't we update the nightlies only after the release happened
Also, why they are still point to 1.12 for the most part, while nightly versions (for desktop at least) has clearly been 1.13.0.dev for the last 2+ month see https://download.pytorch.org/whl/nightly/cpu/torch/

**Pull request closed without merging**